libraries/botbuilder-schema/botbuilder/schema/teams/_models_py3.py

```diff
@@ -2058,3 +2086,265 @@ def __init__(
         self.user = user
         self.meeting = meeting
         self.conversation = conversation
+
+
+class TabContext(Model):
+    """
+    Current tab request context, i.e., the current theme.
+
+    :param theme: Gets or sets the current user's theme.
+    :type theme: str
+    """
+
+    _attribute_map = {
+        "theme": {"key": "theme", "type": "str"},
+    }
+
+    def __init__(self, *, theme=None, **kwargs) -> None:
+        super(TabContext, self).__init__(**kwargs)
+        self.theme = theme
+        self._custom_init()
+
+    def _custom_init(self):
+        return
+
+
+class TabRequest(Model):
+    """
+    Invoke ('tab/fetch') request value payload.
+
+    :param tab_entity_context: Gets or sets current tab entity request context.
+    :type tab_entity_context:
+     ~botframework.connector.teams.models.TabEntityContext
+    :param context: Gets or sets current tab entity request context.
+    :type context:
+     ~botframework.connector.teams.models.TabContext
+    :param state: Gets or sets state, which is the magic code for OAuth Flow.
+    :type state: str
+    """
+
+    _attribute_map = {
+        "tab_entity_context": {"key": "tabEntityContext", "type": "TabEntityContext"},
```

Comment:
Will this look for the JSON property, `tabContext`? See [.NET](https://github.com/microsoft/botbuilder-dotnet/blob/cb33e75b72735e25177fb82084813dcaeadf16c7/libraries/Microsoft.Bot.Schema/Teams/TabRequest.cs#L27)

Reply:
Fixed